platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/attributes/ClientAttributes.kt

```diff
@@ -47,8 +48,9 @@ internal class ClientAttributes(
     @Suppress("SwallowedException")
     private val packageInfo: PackageInfo? by lazy {
         try {
-            context.packageManager.getPackageInfoCompat(appId)
-        } catch (e: Exception) {
+            // we intentionally not use appId+suffix to retain the original's packageInfo
+            context.packageManager.getPackageInfoCompat(context.packageName.orEmpty())
```

Comment:
@pyricau I opted for keeping the `app_version` of the original `app_id` (sans suffix), let me know what you think